pkg/utils/etcdutil/etcdutil.go

```diff
@@ -216,11 +216,13 @@ func createEtcdClient(tlsConfig *tls.Config, acUrls []url.URL) (*clientv3.Client
 		autoSyncInterval = 10 * time.Millisecond
 	})
 	client, err := clientv3.New(clientv3.Config{
-		Endpoints:        endpoints,
-		DialTimeout:      defaultEtcdClientTimeout,
-		AutoSyncInterval: autoSyncInterval,
-		TLS:              tlsConfig,
-		LogConfig:        &lgc,
+		Endpoints:            endpoints,
+		DialTimeout:          defaultEtcdClientTimeout,
+		AutoSyncInterval:     autoSyncInterval,
+		TLS:                  tlsConfig,
+		LogConfig:            &lgc,
+		DialKeepAliveTime:    10 * time.Second,
```

Comment:
Does it have the config item in before?

Reply:
add these configs to avoid failed endponit cannot switched, refer to https://github.com/etcd-io/etcd/issues/7941#issuecomment-334979907

Reply:
The value size is refered to https://github.com/tikv/pd/blob/master/server/region_syncer/client.go#L38-L39